README.md

```
@@ -21,7 +21,7 @@ Get instant, educational feedback from Bad Buggy with transparent cost tracking
 Create `.github/workflows/ai-review.yml`:
 
 ```yaml
-name: Bad Buggy Code Review
+name: 🐙⚡ Pulpo Bob Code Review
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)